src/lib/types/model.ts

```diff
@@ -53,6 +54,7 @@ export interface FeatureToggleDTO {
 export interface FeatureToggle extends FeatureToggleDTO {
     project: string;
     lastSeenAt?: Date;
+    variants?: IVariant[];
```

Comment:
Seems like `variants` should already be here, from `FeatureToggleDTO`.

Reply:
Done